gradle/antlr.gradle

```diff
@@ -29,3 +30,14 @@ sourceSets.configureEach {
     var generateGrammarSource = tasks.named(getTaskName("generate", "GrammarSource"))
     java.srcDir(generateGrammarSource.map { files() })
 }
+
+generateGrammarSource.doLast {
+    final source = layout.buildDirectory.dir('generated-src/antlr/main/com/apple/foundationdb/relational/generated/')
+    final tokensFile = "*.tokens"
+    final dest = layout.projectDirectory.dir('src/main/antlr')
+    copy {
+        from source
+        include tokensFile
+        into dest
+    }
+}
```

Comment:
It would be nice if `clean` could be updated to delete the generated `tokens` files as well. I think you can do that with a new task named something like `cleanGeneratedTokens` of type `Delete` that deletes the token files, and then `clean` can be made to depend on that task.

Reply:
Good idea, done.